pyEpiabm/pyEpiabm/tests/test_sweep/test_spatial_sweep.py

```diff
@@ -2,19 +2,21 @@
 from unittest import mock
 from queue import Queue
 
-import pyEpiabm as pe
+from pyEpiabm.core import Population, Parameters
+from pyEpiabm.property import InfectionStatus
+from pyEpiabm.sweep import SpatialSweep
```

Comment:
Not a major thing, but it is very fine to import the whole of pyEpiabm in the tests (and probably a good thing to do at least some of the time to check everything can be accessed in the expected way :)